From 9ab11ef54c378772982ef65cba3ea6718942899c Mon Sep 17 00:00:00 2001
From: Mark Reynolds <mreynolds@redhat.com>
Date: Tue, 15 Jan 2019 13:55:18 -0500
Subject: [PATCH] Ticket 49540 - FIx compiler warning in ldif2ldbm

https://pagure.io/389-ds-base/issue/49540

Reviewed by: mreynolds(one line commit rule)

(cherry picked from commit 58be90b8bf96a7a0e10740b122035ea03fa13e0f)
(cherry picked from commit c9580477ffe22a08c0094378e81a6927d0dc4ffc)
---
 ldap/servers/slapd/back-ldbm/ldif2ldbm.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ldap/servers/slapd/back-ldbm/ldif2ldbm.c b/ldap/servers/slapd/back-ldbm/ldif2ldbm.c
index 69a2af9cf..11c020af0 100644
--- a/ldap/servers/slapd/back-ldbm/ldif2ldbm.c
+++ b/ldap/servers/slapd/back-ldbm/ldif2ldbm.c
@@ -713,7 +713,7 @@ ldbm_back_ldif2ldbm(Slapi_PBlock *pb)
         uint64_t refcnt;
         refcnt = slapi_counter_get_value(inst->inst_ref_count);
         if (refcnt > 0) {
-            slapi_log_err(SLAPI_LOG_ERR, "ldbm_back_ldif2ldbm", "ldbm: '%s' there are %d pending operation(s)."
+            slapi_log_err(SLAPI_LOG_ERR, "ldbm_back_ldif2ldbm", "ldbm: '%s' there are %" PRIu64 " pending operation(s)."
                     " Import can not proceed until they are completed.\n",
                     inst->inst_name,
                     refcnt);
-- 
2.17.2

